src/containers/access/src/sagas/submitAccessRequest.js

```diff
@@ -86,9 +93,23 @@ function* submitAccessRequestWorker(action :SequenceAction) :Saga<WorkerResponse
     );
 
     if (submitAccessResponse.error) throw submitAccessResponse.error;
-    response = { data: submitAccessResponse.data };
-
-    yield put(submitAccessRequest.success(action.id));
+    const localEntity = Map({
+      // $FlowFixMe invalid-computed-prop
+      [TYPE]: [type || 'Common Application'],
```

Comment:
No change required. `|| 'Common Application'` doesn't make sense as a default in the future.